website/docs/r/vapp_vm.html.markdown

```diff
@@ -152,6 +174,7 @@ example for usage details. **Deprecates**: `network_name`, `ip`, `vapp_network_n
 * `guest_properties` - (Optional; *v2.5+*) Key value map of guest properties
 * `description`  - (Computed; *v2.6+*) The VM description. Note: description is read only. Currently, this field has
   the description of the OVA used to create the VM
+* `override_template_disk` - (Optional; *v2.7+*) Allows to update internal disk in template before first VM boot. Disk are matched by `bus_type`, `bus_number` and `unit_number`. See [Override template Disk](#override_template_disk) below for details.
```

Comment:
`See [Override template Disk](#override_template_disk)` anchor link doesn't work (brings back to this bullet point as opposed to the section below).

Reply:
works for me

Reply:
change, now should work for you too